src/Nix/Builtins.hs

```diff
@@ -170,7 +170,7 @@ builtinsList = sequence
     -- This is compiled in so that we only parse and evaluate it once, at
     -- compile-time.
   , add0 TopLevel "derivation" $(do
-      let Success expr = parseNixText [i|
+      let Success expr = parseNixText [text|
```

Comment:
Note that this change will result in less leading whitespace in the interpolated text. I don't think it hurts in this case though.